### ISSUE TITLE
[CHORE] Ceramic logs flowing into DataDog

### ISSUE THREAD

**Issue by kammerdiener** [gitcoinco/dPopp#278]

**Comment by kammerdiener** [comment]
Significant amounts of research done into this. Trying to ensure that it is flowing properly.

**Comment by kammerdiener** [comment]
Made some progress on this. Need to work on ensuring the Lambda is operating properly in Staging so that things are flowing. Continuing to dig into this.

**Comment by nutrina** [comment]
I don't think this is a priority any more. I would say having the ceramic logs in AWS is sufficient for the now, especially given the fact that we use our own DB as primary storage.
